CHANGELOG.md

```diff
@@ -1,24 +1,27 @@
 # Changelog and versioning
 
 ## Versioning Policy
-
-This project follows [semver](https://semver.org/), with three additions:
-
-- Semver says that major version `0` can include breaking changes at any time. Still, it is common practice to assume that only `0.x` releases (minor versions) can contain breaking changes while `0.x.y` releases (patch versions) are used for backwards-compatible changes (bugfixes and features). This project also follows that practice.
+We decided to use [semver](https://semver.org/) versioning from now on for the SDK. with two additions:
 
 - All undocumented APIs are considered internal. They are not part of this contract.
-
 - Certain features (e.g. integrations) may be explicitly called out as "experimental" or "unstable" in the documentation. They come with their own versioning policy described in the documentation.
 
-We recommend to pin your version requirements against `0.x.*` or `0.x.y`.
-Either one of the following is fine:
 
-```
-sentry-sdk>=0.10.0,<0.11.0
-sentry-sdk==0.10.1
-```
+## 1.0.0
+
+The bump to 1.0 contains a breaking change.
 
-A major release `N` implies the previous release `N-1` will no longer receive updates. We generally do not backport bugfixes to older versions unless they are security relevant. However, feel free to ask for backports of specific commits on the bugtracker.
+- Fix for header extraction for AWS lambda/API extractio
```

Comment:
Is this the breaking change?

Also, is this a typo `extractio`?

Reply:
Yep it's a typo should be extraction*

And nope this is not the breaking change but you are right its not clear which is the breaking change.
So I changed it and added a prefix to breaking changes

So I am just stating underneath release 1.0.0 
That this bump  to 1.0 will have a breaking change
and I prefixed the breaking change with the "BREAKING CHANGE" prefix

Is it not clear?

Reply:
👍